src/tests/src/solver/optim-model-filler/test_readLinearExpressionVisitor.cpp

```diff
@@ -189,12 +192,12 @@ BOOST_FIXTURE_TEST_CASE(visit_complex_expression, MyDummyFixture)
                                           build_context_parameter_with("param2", "8.")},
                                          {},
                                          data);
-    ReadLinearExpressionVisitor visitor(evaluation_context, {0, 0});
+    ReadLinearExpressionVisitor visitor(evaluation_context, {0, 0}, componentId);
     auto linear_expression = visitor.dispatch(big_sum).GetLinearExpressions().at(0);
     BOOST_CHECK_EQUAL(linear_expression.offset(), 10.);
     BOOST_CHECK_EQUAL(linear_expression.coefPerVar().size(), 2);
-    BOOST_CHECK_EQUAL(linear_expression.coefPerVar()["var1"], 4.);
-    BOOST_CHECK_EQUAL(linear_expression.coefPerVar()["var2"], 6.);
+    BOOST_CHECK_EQUAL(linear_expression.coefPerVar().at(FullKey(componentId, "var1", 0, 0)), 4.);
```

Comment:
`coefPerVar(FullKey(/* ... */))` would look nicer

Reply:
yes but this overload would be used only for test?

Reply:
right, so let's not do that